pkg/aws/common_validator_test.go

```diff
@@ -303,3 +306,46 @@ func TestValidateLabelValues(t *testing.T) {
 		})
 	}
 }
+
+func Test_OrganizationNotExists(t *testing.T) {
+	organization := &securityv1alpha1.Organization{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "giantswarm",
+		},
+		Spec: securityv1alpha1.OrganizationSpec{},
+	}
+
+	ctx := context.Background()
+
+	fakeK8sClient := unittest.FakeK8sClient()
+	err := fakeK8sClient.CtrlClient().Create(ctx, organization)
+	if err != nil {
+		panic(err)
+	}
+
+	// if organization is empty it should fail
+	err = ValidateOrganizationLabelContainsExistingOrganization(ctx, fakeK8sClient.CtrlClient(), unittest.DefaultClusterEmptyOrganization())
+	if !IsOrganizationNotFoundError(err) {
+		t.Fatalf("it should fail when using a non existing Organization")
+	}
+
+	// if organization label is not given it should fail
+	err = ValidateOrganizationLabelContainsExistingOrganization(ctx, fakeK8sClient.CtrlClient(), unittest.DefaultClusterWithoutOrganizationLabel())
+	if !IsOrganizationLabelNotFoundError(err) {
+		t.Fatalf("it should fail when using Organization label is not given")
+	}
+}
+
+func Test_OrganizationExists(t *testing.T) {
```

Comment:
I am not blocking this if it does the job but I'm wondering why we are using two separate tests for `ValidateOrganizationLabelContainsExistingOrganization` instead of going with a table driven test calling that function with different inputs and expectations.

Reply:
True it was quite late 🤣 . Good point I'll change that 👍🏻